apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/InsertOptions.cs

```diff
@@ -30,12 +30,48 @@ public sealed class InsertOptions
         /// </summary>
         public bool? AllowUnknownFields { get; set; }
 
+        /// <summary>
+        /// Insert all valid rows of a request, even if invalid rows exist. The
+        /// default value is false, which causes the entire request to fail if any invalid
```

Comment:
I wouldn't say the default is false - I'd just say what the default behaviour is. (Because the default value of this property is null rather than false.)

Reply:
Done.